content/en/logs/processing/_index.md

```diff
@@ -72,9 +72,36 @@ If you want to learn more about pure parsing possibilities of the Datadog applic
 For optimal usage of the Log Management solution, Datadog recommends using at most 20 processors per pipeline and 10 parsing rules within a grok processor. 
 Datadog reserves the right to disable underperforming parsing rules, processors, or pipelines that might impact Datadog's service performance.
 
-## Reserved attributes
+## JSON Logs Preprocessing
+
+JSON Logs preprocessing comes with a default configuration that work for standard log forwarders. Edit this configuration at any time to adapt to custom or specifc log forwarding approaches. To change the default values, go to the [Configuration page][5] and edit the `JSON Logs Preprocessing`:
+
+{{< img src="logs/processing/json_logs_preprocessing.gif" alt="JSON Logs Preprocessing Tile"  style="width:80%;">}}
```

Comment:
I don't get why this one is broken in preview (although all CI tests passed).

+ The alternative text is still Reserved Attribute Tile 🤔 , which is the alternative text for the image we had in the previous version.